group.py

```diff
@@ -1,5 +1,10 @@
-"""An example of how to represent a group of acquaintances in Python."""
+# - Jill is 26, a biologist and she is Zalika's friend and John's partner.
+# - Zalika is 28, an artist, and Jill's friend
+# - John is 27, a writer, and Jill's partner.
+# - Nash is 34, a chef, John's cousin and Zalika's landlord.
 
-# Your code to go here...
+group_dict=[{'name':'Jill','age':26,'job':'biologist','connection':{'Zalika':'friend','John':'partner'}},
```

Comment:
You could possibly use a new line for the connection to more easily read the code